client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/task-helper.ts

```diff
@@ -90,45 +90,4 @@ describe( 'Task Helpers', () => {
 			} );
 		} );
 	} );
-
-	describe( 'filterDomainUpsellTask', () => {
-		describe( 'when site plan is free', () => {
-			it( 'return original enchanceTasks', () => {
-				const task = buildTask( {
-					id: 'domain_upsell',
-					completed: false,
-					disabled: true,
-					taskType: 'blog',
-					title: 'domain upsell task',
-				} );
-				const tasks = [ task ];
-				const site = buildSiteDetails( { plan: { is_free: true } } );
-				// domain_upsell is still in the array
-				expect(
-					filterDomainUpsellTask( tasks, site )?.findIndex(
-						( task ) => ( task.id = 'domain_upsell' )
-					)
-				).toBe( 0 );
-			} );
```

Comment:
Why remove the test here?

The 'when site plan is free' seems like it should still be relevant?  The 'when site plan is not free' below may need updating - im fine with removing that item for now and updating in an immediate follow up.